contrib/bpftrace/gc_all.bt

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bpftrace
+
+BEGIN
+{
+	printf("Tracing Julia GC Times... Hit Ctrl-C to end.\n");
+}
+
+usdt:usr/lib/libjulia-internal.so:julia:gc__begin
+{
+	$now = nsecs;
+	@time[pid] = $now;
+	@start[pid] = $now;
+}
+
+usdt:usr/lib/libjulia-internal.so:julia:gc__stop_the_world
+/@start[pid]/
+{
+	$now = nsecs;
+	@stop_the_world[pid] = hist(($now - @time[pid]) / 1000);
+	@time[pid] = $now;
+}
+
+usdt:usr/lib/libjulia-internal.so:julia:gc__end
+/@start[pid]/
+{
+	$now = nsecs;
+	@gc_total[pid] = hist(($now - @start[pid]) / 1000);
+	@gc_phase[pid] = hist(($now - @time[pid]) / 1000);
+	@time[pid] = $now;
+    delete(@start[pid]);
```

Comment:
I'm guessing the indentation difference is conventional, given the same difference below? :)